packages/viem/src/__tests__/index-test.ts

```diff
@@ -1,15 +1,48 @@
-import { WalletClient, createWalletClient, http } from "viem";
-import { sepolia } from "viem/chains";
+import { TurnkeyActivityError } from "@turnkey/http";
+import {
+  type LocalAccount,
+  type Chain,
+  type Hex,
+  createTestClient,
+  publicActions,
+  walletActions,
+  http,
+  stringToHex,
+  getContract,
+  getContractAddress,
+  parseEther,
+  parseUnits,
+} from "viem";
+import { foundry } from "viem/chains";
+import { TurnkeyClient } from "@turnkey/http";
+import { ApiKeyStamper } from "@turnkey/api-key-stamper";
+import { createAccount } from "../";
+import Test721 from "./contracts/artifacts/src/__tests__/contracts/source/Test721.sol/Test721.json";
+import { expect, beforeEach, describe, test } from "@jest/globals";
 
-import { createApiKeyAccount } from "../";
+// @ts-expect-error
+const testCase: typeof test = (...argList) => {
+  if (!process.env.BANNED_TO_ADDRESS) {
+    // For now, this test requires certain environment variables to be injected (from Turnkey's internal environment)
+    return test.skip(...argList);
+  }
```

Comment:
This means the test won't be run in CI right? I don't think we inject this env variable anywhere right now.

Reply:
It's injected here: https://github.com/tkhq/mono/blob/2fac89233c77dfd86d64094537703b31bd4271eb/src/go/e2e/e2e_viem_test.go#L103

Reply:
right right. But I think we should run this in _this repo_'s CI as well no? That would be useful to detect bad changes to the viem package itself without running tests through our private monorepo.

Reply:
ah i see what you're saying. i'll mess around with CI!

Reply:
If it's too hard don't worry about it though: can be its own issue/PR. Not running tests in the status quo (unfortunately); ethers tests are in the same boat!

https://github.com/tkhq/sdk/blob/397b6b5fb1a17d5a88c372e97c1485c9abacb419/packages/ethers/src/__tests__/index-test.ts#L15